mmdet/core/visualization/image.py

```diff
@@ -0,0 +1,156 @@
+import os.path as osp
+import warnings
+
+import matplotlib.pyplot as plt
+import mmcv
+import numpy as np
+from matplotlib.collections import PatchCollection
+from matplotlib.patches import Polygon
+
+
+def color_val_matplotlib(color):
+    """Convert opencv color value to matplotlib (BGR->RGB->Norm)"""
+    color = mmcv.color_val(color)
+    color = [color / 255 for color in color[::-1]]
+    return tuple(color)
+
+
+def imshow_det_bboxes(img,
+                      bboxes,
+                      labels,
+                      segms=None,
+                      class_names=None,
+                      score_thr=0,
+                      bbox_color='green',
+                      text_color='green',
+                      mask_color='green',
+                      thickness=2,
+                      font_scale=0.5,
+                      font_size=13,
+                      win_name='',
+                      fig_size=(15, 10),
+                      show=True,
+                      wait_time=0,
+                      out_file=None):
+    """Draw bboxes and class labels (with scores) on an image.
+
+    Args:
+        img (str or ndarray): The image to be displayed.
+        bboxes (ndarray): Bounding boxes (with scores), shaped (n, 4) or
+            (n, 5).
+        labels (ndarray): Labels of bboxes.
+        segms (ndarray or None): Masks, shaped (n,h,w) or None
+        class_names (list[str]): Names of each classes.
+        score_thr (float): Minimum score of bboxes to be shown.  Default: 0
+        bbox_color (str or BGR tuple (int) or :obj:`Color`):
```

Comment:
This is only about type. So we should keep tuple(int) and add the order of color in the description like:
```python

bbox_color (str or tuple(int) or :obj:`Color`): Color of bbox lines. The tuple of color should be in BGR order. Default: 'green'
```